cloud/amazon/route53.py

```diff
@@ -342,6 +342,10 @@ def main():
     if record_in[-1:] != '.':
         record_in += "."
 
+    if failover_in!=None:
+	if failover_in not in [ 'PRIMARY' , 'SECONDARY' ]:
+          module.fail_json(msg= "For failover only PRIMARY and SECONDARY values are allowed")
+
```

Comment:
instead of doing your own validation, why just not just use the 'choices' facility in the built in module spec?